rust_dev_preview/dynamodb/README.md

```diff
@@ -22,26 +36,24 @@ DynamoDB is a fully managed NoSQL database service that provides fast and predic
 - [Lists your tables](src/bin/list-tables.rs) (ListTables)
 - [Lists your local tables](src/bin/list-tables-local.rs) (ListTables)
 - [Minimal version of listing tables](src/bin/list-tables-main.rs) (ListTables)
-- [Create a table, adds some items from a file to the table, queries the table, and deletes the table](src/bin/movies.rs) (CreateTable, DeleteTable, ListTables, PutItem, Query)
 
-## ⚠ Important
+### Scenarios
 
-- We recommend that you grant this code least privilege, 
-  or at most the minimum permissions required to perform the task.
-  For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions.
-  Some AWS services are available only in specific
-  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account.
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Create a table, adds some items from a file to the table, queries the table, and deletes the table](src/bin/movies.rs) (CreateTable, DeleteTable, ListTables, PutItem, Query)
+  - To run this example, you should replace the `moviedata.json` file with the [DynamoDB Movies](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/samples/moviedata.zip) zip database.
+  - On a \*nix environment with bash, you can use this command (be sure to run it from the same directory as this README):
```

Comment:
- On a \*nix environment with bash, you can use the following command. (Be sure to run it from the same directory as this README.)